app/templates/client/index.html

```diff
@@ -50,7 +50,11 @@
       <script src="socket.io-client/socket.io.js"></script><% } %>
     <!-- endbuild -->
 
+        <% if(filters.babel) { %>
+        <!-- build:js(.tmp) app/app.js -->
```

Comment:
Heya - bit of a slow reply! Babel is precompiling all the /client js resources to /.tmp. 

If we inject all the js resources from /.tmp and /client with the non-babel index.html annotation  .... <!-- build:js({.tmp,client}) app/app.js -->  ..... then we get a double up of all ES5 and ES6 items causing angular to die with multiple definitions of modules or something along those lines.